### PR TITLE
[CARBONDATA-2135] Documentation for Table comment and Column Comment

### PR DESCRIPTION
**Pull request by sgururajshetty**
Documentation for table comment and column comment

### CODE REVIEW

**Comment by ravipesala** [comment]
Can one of the admins verify this patch?

**Comment by CarbonDataQA** [comment]
Can one of the admins verify this patch?

**Comment by CarbonDataQA** [comment]
Can one of the admins verify this patch?

**Comment by sgururajshetty** [comment]
@chenliang613 kindly review

File: docs/data-management-on-carbondata.md [hunk 1]
```diff
@@ -286,7 +286,40 @@ This tutorial is going to introduce all commands and data operations on CarbonDa
   * If the table is aggregate table, then all the aggregate tables should be copied to the new database location.
   * For old store, the time zone of the source and destination cluster should be same.
   * If old cluster used HIVE meta store to store schema, refresh will not work as schema file does not exist in file system.
+
+### Table and Column Comment
+
+  You can provide more information on table by using table comment; similarly you can provide more information about a particular column using column comment.
```
Comment (manishgupta88) [inline docs/data-management-on-carbondata.md]: Use dot in place of semicolon

**Comment by manishgupta88** [comment]
LGTM

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.2.1, Please check CI http://88.99.58.216:8080/job/ApacheCarbonPRBuilder/2623/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/3868/

**Comment by ravipesala** [comment]
SDV Build Success , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/3649/

**Pull request closed without merging**